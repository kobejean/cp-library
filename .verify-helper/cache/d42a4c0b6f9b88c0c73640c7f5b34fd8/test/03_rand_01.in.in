50
2 128
186 4
156 141
174 193
181 94
10 102
105 45
55 15
160 74
10 195
135 79
176 77
55 73
196 0
55 117
123 118
55 17
7 10
80 130
129 149
137 40
95 130
54 199
116 70
183 53
148 94
130 119
187 170
172 112
7 159
153 73
186 158
68 2
45 26
55 161
153 195
106 89
47 60
131 124
0 19
143 44
152 183
70 165
6 61
185 142
74 82
110 105
132 55
176 17
173 187
20
143 243 29 129
15 115 189 289
9 109 188 288
154 254 43 143
101 201 71 171
116 216 164 264
77 177 112 212
5 105 11 111
12 112 182 282
92 192 123 223
1 101 124 224
4 104 37 137
2 102 4 104
163 263 191 291
73 173 175 275
106 206 159 259
26 126 96 196
59 159 163 263
158 258 177 277
84 184 147 247
