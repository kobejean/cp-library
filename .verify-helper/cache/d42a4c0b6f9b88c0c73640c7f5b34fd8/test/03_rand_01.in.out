4
8
11
24
25
28
30
40

22

9
22

2
4
8
11
24
28
44

2
4
8
10
11
15
19
25
26
27
28
30
31
36
38
44
46

3
27
35
41
49

2
3
15
18
19
21
26
28
35
38
41
49

5
6
7
12
16
33
37
43
45

22

2
3
19
21
27
31
35
38
41
44
49

0
9
18
21
22
29
34
42

5
12
14
18
21
37
43
45

5
7
12
16
17
33
37
43
45

3

35
41
49

3
27
35
41
49

14
15
18
21
34
42
46

35
41
42

3
49

3
19
35
41
49

